packages/adapters/src/adapters/helpers/toScoordType.ts

```diff
@@ -0,0 +1,16 @@
+import type { Types } from "@cornerstonejs/core";
+
+export function toScoord(
+    { worldToImageCoords, is3DMeasurement, referencedImageId },
+    point
+) {
+    if (is3DMeasurement) {
+        return { x: point[0], y: point[1], z: point[2] };
+    }
+    const point2 = worldToImageCoords(referencedImageId, point);
```

Comment:
why are we passing worldToImageCoords here? this can get imported from cornerstone since it is a utility